tasks/run-collectors/run-collectors.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: run-collectors
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to run collectors defined in the ReleasePlan and ReleasePlanAdmission
+  params:
+    - name: releasePlanAdmissionPath
+      type: string
+      description: Path to the JSON string of the ReleasePlanAdmission in the data workspace
+    - name: releasePlanPath
+      type: string
+      description: Path to the JSON string of the ReleasePlan in the data workspace
+    - name: resultsDir
+      type: string
+      description: The relative path in the workspace to save the collector results to
+  workspaces:
+    - name: data
+      description: Workspace where the CRs are stored
+  steps:
+    - name: run-collectors
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set -xeo pipefail
+
+        DEFAULT_TIMEOUT=900
+        COLLECTORS_REPOSITORY="https://github.com/SOME_ORG/SOME_REPO"
```

Comment:
What's this? So all the collectors will be in one repo that we'll maintain? You wanna merge it with this example value and change later? In that case at least add like a TODO comment or something?

Reply:
From the Jira
`This task will assume that collectors are Python scripts  defined in a managed collectors repository`
I will add a TODO

Reply:
comment added